.gitignore

```diff
@@ -1 +1,4 @@
-.env
\ No newline at end of file
+.env
+.anonymized_text
+.__pycache__/
+..pytest_cache
```

Comment:
`.pytest_cache` instead of `..pytest_cache`

Reply:
here I did .. cuz already the name of pytest_cache starts with .pytest_cache and it worked!